hrjobcontract/hrjobcontract.php

```diff
@@ -278,10 +278,10 @@ function hrjobcontract_civicrm_buildForm($formName, &$form) {
  * to give some room for other extensions to place
  * their tabs between these two.
  */
-function hrjobcontract_civicrm_tabs(&$tabs) {
+function hrjobcontract_civicrm_tabs(&$tabs, $contactId) {
```

Comment:
Just a note: The _tabs hook has been deprecated in civicrm 4.7 (https://wiki.civicrm.org/confluence/pages/viewpage.action?pageId=86213406) and _tabSet should be used instead. Since we use _tabs in a lot of other extensions, I think we need a separate ticket to update them all at once.